### PR TITLE
Master nutty modify framework newwidget

### PR DESCRIPTION
**Pull request by nutty898**

### CODE REVIEW

File: extensions/CocoStudio/Action/CCActionNode.cpp [hunk 1]
```diff
@@ -159,10 +159,10 @@ void ActionNode::initActionNodeFromRoot(CCObject* root)
 	}
 	else
 	{
-		UIWidget* rootWidget = dynamic_cast<UIWidget*>(root);
+        cocos2d::gui::Widget* rootWidget = dynamic_cast<cocos2d::gui::Widget*>(root);
```
Comment (minggo) [inline extensions/CocoStudio/Action/CCActionNode.cpp:158]: Wrong indent.

Comment (nutty898) [inline extensions/CocoStudio/Action/CCActionNode.cpp:158]: OK, I'll fix that.

**Comment by minggo** [comment]
Compiling error https://travis-ci.org/cocos2d/cocos2d-x/builds/15980621.

File: extensions/CocoStudio/GUI/UIWidgets/UITextField.cpp [hunk 2]
```diff
@@ -1,671 +1,596 @@
...
+{
+    _passwordEnabled = enable;
+}
+
+bool UICCTextField::isPasswordEnabled()
+{
+    return _passwordEnabled;
+}
+
+void UICCTextField::setPasswordStyleText(const char* styleText)
```
Comment (minggo) [inline extensions/CocoStudio/GUI/UIWidgets/UITextField.cpp:208]: Why this class name is "UICC" prefix.

File: samples/Cpp/TestCpp/Classes/ExtensionsTest/CocoStudioGUITest/CocosGUIExamplesScene.h [hunk 3]
```diff
@@ -1,135 +1,135 @@
...
+//    
+//    CCPoint movePoint;
+//    CCPoint lastPoint;
+//    
+//    CCPoint widgetLastWorldSpace;
+//    CCPoint widgetLastNodeSpace;
+//    UIWidget* lastWidgetParent;
+//    //
+//};
+//
```
Comment (minggo) [inline samples/Cpp/TestCpp/Classes/ExtensionsTest/CocoStudioGUITest/CocosGUIExamplesScene.h]: Please remove unneeded codes.

File: samples/Cpp/TestCpp/Classes/ExtensionsTest/CocoStudioGUITest/CocosGUIScene.cpp [hunk 4]
```diff
@@ -70,17 +70,17 @@ void CocosGUITestScene::menuCallback(CCObject *pSender)
...
             break;
             
         case 1:
         {
-            CocosGUIExamplesRegisterScene* pScene = new CocosGUIExamplesRegisterScene();
-            pScene->runThisTest();
-            pScene->release();
+//            CocosGUIExamplesRegisterScene* pScene = new CocosGUIExamplesRegisterScene();
+//            pScene->runThisTest();
+//            pScene->release();
```
Comment (minggo) [inline samples/Cpp/TestCpp/Classes/ExtensionsTest/CocoStudioGUITest/CocosGUIScene.cpp]: Please remove unneeded codes.

**Comment by minggo** [comment]
And please modify all sample project settings in `samples` and `template`.

File: extensions/CocoStudio/GUI/UIWidgets/UITextField.cpp (hunk 2 above)
Comment (nutty898) [inline extensions/CocoStudio/GUI/UIWidgets/UITextField.cpp:208]: That is a class which developers need not to use, it just for GUI framework.

File: samples/Cpp/TestCpp/Classes/ExtensionsTest/CocoStudioGUITest/CocosGUIExamplesScene.h (hunk 3 above)
Comment (nutty898) [inline samples/Cpp/TestCpp/Classes/ExtensionsTest/CocoStudioGUITest/CocosGUIExamplesScene.h]: I'll update the new samples later.

**Pull request merged**